src/main/java/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Colors.values().length);
+        Colors c = Colors.values()[index];
+        String r = c.toString();
+        return r;
```

Comment:
```suggestion
        return  c.toString();
```
краще одразу повертати результат